### ISSUE TITLE
`pairing` error is not handled after user creates a custom pairing code

### ISSUE THREAD

**Issue by saledjenic** [status-im/status-desktop#8065]
When user creates a custom pairing code (which is not the case for default pairing code) any flow may ask for the pairing code if there is no `pairings.json` file stored locally (keycard will ask for that only once). What we need to do in that case is to ask user to enter his pairing code (which is another level of security) and resume the current flow using `pairing-pass` as key and entered code as value. 

We don't have that screen in figma (guess it will look like enter password screen).

**Comment by saledjenic** [comment]
@alaibe as we discussed on the offsite, this should be very low priority issue, most likely after MVP thing. Giving that `Create a new pairing code` option from here https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=3589%3A157439 will be most likely temporary disabled. Not 100% sure, but that's something John mentioned. 

All in all we already have that flow developed, we just need a new screen as described in this issue and handling `pairing-pass` response for all possible flows.
